### ISSUE TITLE
Missing EventEmitter on recent RCs of @angular/cli 

### ISSUE THREAD

**Issue by kamilkisiela** [apollographql/subscriptions-transport-ws#99]
An error:
> eventemitter3_1.EventEmitter is not a constructor

It happens inside a project that has been created by `@angular/cli@^1.0.0-rc.1`.

---

In [client.ts](https://github.com/apollographql/subscriptions-transport-ws/blob/b0c71084e88c11ddfb8cefb1d7996e15507bf808/src/client.ts#L2), we've got this:

```ts
import { EventEmitter } from 'eventemitter3';
```

In [eventemitter3/index.js](https://github.com/primus/eventemitter3/blob/e5ba1e240edfb4f3a4177c592ce77cb949b9aadb/index.js#L304):

```ts
EventEmitter.EventEmitter = EventEmitter;
export default EventEmitter;
```

Webpack creates a bundle file that contains that:

```js
var _eventemitter3 = require('eventemitter3');

// inside SubscriptionClient of `subscriptions-transport-ws`

this.eventEmitter = new _eventemitter3.EventEmitter();
```

The thing is, `_eventemitter3` has no `EventEmitter` property assigned. It's missing...

```js
console.log(_eventemitter3); // EventEmitter class
console.log(_eventemitter3.EventEmitter); // undefined
```

When you look up for the `eventemitter3` package inside the bundle file, there's no `EventEmitter.EventEmitter` but there's another property available that has been assigned the same way (`EventEmitter.prefixed`). It somehow removes the assignment from the code.

Weird...

**Comment by kamilkisiela** [comment]
Continuing apollographql/GitHunt-Angular#100

**Comment by jmarlow4** [comment]
Any progress on this?

**Comment by jhartma** [comment]
having the exact same issue

**Comment by tobiaghiraldini** [comment]
Same issue also here. even with subscriptions-trasport-ws@0.5.5. I also tried to upgrade the eventemitter3 dependency with no luck.
this issue is preventing to setup a ws communication with opengraph servers. Is there any workaround at least?

**Comment by nakedcity** [comment]
check: npm ls eventemiter3
packages are using the eventemiter3 e.g karma uses 1.x
solution: npm i eventemiter3 --save

**Comment by jhartma** [comment]
thanks, that works

**Comment by amsterdam80** [comment]
you welcome

**Comment by Urigo** [comment]
Fixed in `0.6.0`.
